### PR TITLE
Visited links for LW

### PR DESCRIPTION
**Pull request by Raemon**
PR to give LW and AF visited-link-color

Here are the current set of colors that Oli gave:

image

For comparison, here's the original:

image

I think the new, darker link color is actually somewhat hard to parse and prefer leaving the original.

I also would just really prefer us to use a different color rather than trying to make another shade of green, which is pretty annoying to parse (and also feels sort of... sickly green?)

I personally prefer something closer to this brown, which feels harmonious with the green and the background but a) is more clearly different, b) feels something like a "wholesome healthy earth tone" to me instead of a sickly green.

image

But, can just ship the current version.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1208115889967380) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by jimrandomh** [comment]
Imo those colors are too close; I had a hard time telling the difference even when I was explicitly looking for it.

**Review by oetherington — COMMENTED** [review]
No objections from EAF

**Comment by Discordius** [comment]
I think we could go with the second set of colors and revert or make things more subtle if we don't like it

**Comment by Raemon** [comment]
I've pushed an update with a brownish color. I fiddled around a bit more and currently am using this one (contrast ratio 4.09).

image

image

**Review by Discordius — APPROVED** [review]
